### DISCUSSION TITLE
Loading PGlite fails

### DISCUSSION THREAD

**Discussion by balazskosi** [observablehq/framework#1827] [Q&A]
```ts
import { PGlite } from "npm:@electric-sql/pglite"

const db = await PGlite.create();
```

Results in:

```
Error: Invalid FS bundle size: 87 !== 2987805
```

The preview console writes:

```
npm:@electric-sql/pglite@0.2.13/postgres.data → src/.observablehq/cache/_npm/@electric-sql/pglite@0.2.13/postgres.data
Error: unable to fetch: https://cdn.jsdelivr.net/npm/@electric-sql/pglite@0.2.13/postgres.data
    at file:///home/rebmekop/Documents/mikomi/obs/hello-framework/node_modules/@observablehq/framework/dist/npm.js:71:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
Error: unable to fetch: https://cdn.jsdelivr.net/npm/@electric-sql/pglite@0.2.13/postgres.data
    at file:///home/rebmekop/Documents/mikomi/obs/hello-framework/node_modules/@observablehq/framework/dist/npm.js:71:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

What is right, because `postgres.data` is at `https://cdn.jsdelivr.net/npm/@electric-sql/pglite@0.2.13/dist/postgres.data` not at `https://cdn.jsdelivr.net/npm/@electric-sql/pglite@0.2.13/postgres.data` (note the `/dist`).

When downloading with npm and using on the server side it works 
I've made it work patching the path in the downloaded files. 
It references the file like this: `e=new URL("./postgres.data",import.meta.url);`

Is this a problem with the `pglite` package, with the import code or the way I am importing it?

**Comment by Fil** [comment]
Thanks. I'm not sure how to fix this, but here's another solution, which does not require patching the code:

~~~~md
```js echo
import {PGlite} from "npm:@electric-sql/pglite/dist/index.js"
const db = await PGlite.create();
```

```js
display(await db.query("SELECT 'Hello world' as message;"))
```
~~~~

**Reply by balazskosi** [reply]
Thanks! I've used this (importing `dist/index.js`) until I needed the `fuzzystrmatch` extension.

`import { fuzzystrmatch } from 'npm:@electric-sql/pglite/contrib/fuzzystrmatch'`

Where I failed to make it work loading the extension.
